packages/kbn-tinymath/src/grammar.pegjs

```diff
@@ -0,0 +1,100 @@
+// tinymath parsing grammar
```

Comment:
You didn't do any grammar changes yet, right?

Reply:
Right, this is the same as the previous grammar.